src/SOS/Strike/strike.cpp

```diff
@@ -1417,7 +1446,19 @@ HRESULT PrintObj(TADDR taObj, BOOL bPrintFields = TRUE)
     DacpMethodTableData mtabledata;
     if ((Status=mtabledata.Request(g_sos,objData.MethodTable)) == S_OK)
     {
-        DMLOut("EEClass:     %s\n", DMLClass(mtabledata.Class));
+        BOOL net9preferCanonMT = FALSE;
```

Comment:
nit:  Why not just eliminate this variable and use `PreferCanonMTOverEEClassForNET9` directly below?